app/javascript/mastodon/features/about/index.jsx

```diff
@@ -203,11 +205,16 @@ class About extends PureComponent {
             ))}
           </Section>
 
-          <LinkFooter />
+          <Section title={intl.formatMessage(messages.aboutMastodon)}>
+            <div className='prose'>
+              <p><FormattedMessage id='about.mastodon.version' defaultMessage='{domain} is powered by Mastodon {version}.' values={{ domain, version: server.get('version') }} /></p>
```

Comment:
```suggestion
              <p><FormattedMessage id='about.mastodon.version' defaultMessage='{domain} is powered by the free, open-source software Mastodon ({version}).' values={{ domain, version: server.get('version') }} /></p>
```

Reply:
I think mentioning “free, open-source” once is enough :-)